### DISCUSSION TITLE
Working on multiple issues

### DISCUSSION THREAD

**Discussion by vanshm2703** [sanjay-kv/Open-source-Practice#851] [Q&A] [answered]
Can I work on 2issues simultaneously , from 2 same/different projects?

**Comment by tat-tvam-asii** [comment, marked as answer]
Yes you definitely can work .

**Comment by lovelymahor** [comment]
Yes, you can work on multiple issues simultaneously, whether they are from the same project or different projects. However, it's important to manage your time and workload effectively to ensure you can handle both issues without compromising the quality of your contributions.
